### PR TITLE
Update regression.js

### PR DESCRIPTION
**Pull request by modsushi**
Linear Regression function returns an object, not an array. Previously, it was returning NaN for "y"

### CODE REVIEW

**Comment by virtualstaticvoid** [comment]
@iamjab Thanks.

File: regression.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ function fitData(data, typ) {
 
     ret = linearRegression(x, y);
```
Comment (virtualstaticvoid) [inline regression.js:107]: The `linearRegression` method does return an array.

**Comment by virtualstaticvoid** [comment]
Apologies, I merged this a little to quickly. I've reverted this pull request with #10

File: regression.js (hunk 1 above)
Comment (modsushi) [inline regression.js:107]: My mistake, there was another function with same name overwriting the original in my project.

**Pull request merged**